src/systems/CMakeLists.txt

```diff
@@ -61,12 +61,14 @@ function(gz_add_system system_name)
   # (i.e. libignition-gazebo1-name-system.so), but for portability of SDF
   # files, we also install an unversioned symlink into the same versioned folder.
   set(versioned ${CMAKE_SHARED_LIBRARY_PREFIX}${system_target}${CMAKE_SHARED_LIBRARY_SUFFIX})
-  set(unversioned ${CMAKE_SHARED_LIBRARY_PREFIX}${PROJECT_NAME_NO_VERSION_LOWER}-${system_name}${CMAKE_SHARED_LIBRARY_SUFFIX})
+  set(unversioned_file_name ${PROJECT_NAME_NO_VERSION_LOWER}-${system_name}${CMAKE_SHARED_LIBRARY_SUFFIX})
+  set(unversioned ${CMAKE_SHARED_LIBRARY_PREFIX}${unversioned_file_name})
   if (WIN32)
-    # create_symlink requires cmake 3.13 on windows
-    cmake_minimum_required(VERSION 3.13 FATAL_ERROR)
+    # symlinks on Windows require admin priviledges, fallback to copy
+    install(FILES ${versioned} DESTINATION ${CMAKE_SHARED_LIBRARY_PREFIX} RENAME ${unversioned_file_name})
```

Comment:
Can we install to `IGNITION_GAZEBO_PLUGIN_INSTALL_DIR` instead of `CMAKE_SHARED_LIBRARY_PREFIX`?

Reply:
In fact, in my latest branch I've removed completely the this from Windows since I liked to assume  from comment
`for portability of SDF files, we also install an unversioned symlink into the same versioned folder.` that Windows don't need any portability since it was not working on Windows before. Is this correct?

Reply:
The portability mentioned there is between Ignition versions in the past and future. For example, if we don't have unversioned libraries, this SDF file would need to hardcode the Gazebo version as `ignition-gazebo4-physics-system`:

https://github.com/ignitionrobotics/ign-gazebo/blob/bccea9087dd4355fc6d273d41a191d5c16563175/examples/worlds/empty.sdf#L60-L63

Reply:
Ah thanks. I've modified the code to make it to copy versioned plugins to unversioned in 2a9836f